presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -529,6 +528,21 @@ public void setOutput(Optional<Output> output)
         this.output.set(output);
     }
 
+    private void attachLastDataCommitTimesToOutput(ConnectorCommitHandle commitHandle)
+    {
+        if (output.get().isPresent()) {
+            SchemaTableName table = new SchemaTableName(output.get().get().getSchema(), output.get().get().getTable());
```

Comment:
Can we add 
`Output currentOutput = output.get().get();`
and then use `currentOutput` instead of `output.get().get()` everywhere.